fusionfusion/arg_parser.py

```diff
@@ -16,6 +16,12 @@ def create_parser():
     parser.add_argument("--star", metavar = "star.Chimeric.out.sam", default = None, type = str,
                         help = "the path to the chimeric sam file by STAR")
 
+    parser.add_argument("--star_sj_tab", metavar = "star.SJ.out.tab", default = None, type = str,
+                        help = "the path to the SJ.out.tab file by STAR")
+
+    parser.add_argument("--star_aligned_bam", metavar = "star.Aligned.sortedByCoord.out.bam", default = None, type = str,
+                        help = "the path to the aligned bam file by STAR (which must be sorted by coordinate)")
```

Comment:
https://www.flake8rules.com/rules/E251.html
https://www.python.org/dev/peps/pep-0008/#other-recommendations
```suggestion
    parser.add_argument("--star_sj_tab", metavar="star.SJ.out.tab", default=None, type=str,
                        help="the path to the SJ.out.tab file by STAR")

    parser.add_argument("--star_aligned_bam", metavar="star.Aligned.sortedByCoord.out.bam", default=None, type=str,
                        help="the path to the aligned bam file by STAR (which must be sorted by coordinate)")
```